### ISSUE TITLE
Prompted for auth even with LAB_CORE_TOKEN set 

### ISSUE THREAD

**Issue by jackmac92** [zaquestion/lab#566]
Hey minor bug/inconsistency with the README

I've used this cli for a while, and I've always just used LAB_CORE_TOKEN for auth. After updating, and running `lab repo browse`, I was prompted to create a personal access token.

```Create a token with scope 'api' here: https://gitlab.com/profile/personal_access_tokens
Enter default GitLab token, or leave blank to provide a command to load the token:
Config saved to /home/jmccown/.config/lab/lab.toml
Valid!
```

I was a little confused because my token hasn't expired yet. After entering a token at the prompt, it creates the config file, and now everything works as expected (it still respects LAB_CORE_TOKEN over what is in the config file).

Based on the README, I think I should not get prompted if I have LAB_CORE_TOKEN already set

**Comment by fmuellner** [comment]
Duplicate of #558

**Comment by zaquestion** [comment]
@jackmac92 I've released a new version of `lab` `v0.19.0` with the fix from #558, please let us know if you're still seeing this issue.
